### PR TITLE
Category, sub-category and article titles and icons should be links [YEP-26] [YEP-36]

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

File: tuneme/templates/polls/poll_page.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 		<h2>{% trans "What's your opinion?" %}</h2>
 		<div class="block turq">
 			<div class="title">
-				<div class="icon"><img src="{% static 'img/pie.png' %}" alt="Have Your Say"></div>
+				<div class="icon"><a href=""><img src="{% static 'img/pie.png' %}" alt="Have Your Say"></a></div>
```
Comment (smarzban) [inline tuneme/templates/polls/poll_page.html]: @milton in order to go to the polls results page the URL needs a vote value, is there anyway that we can go to the results page without voting ?

Comment (miltontony) [inline tuneme/templates/polls/poll_page.html]: You can link to the results page `{% url 'polls:results' poll.id %}` I think

Comment (smarzban) [inline tuneme/templates/polls/poll_page.html]: @miltontony  these are the polls URLs

```
/polls/ polls.views.IndexView   index
/polls/<pk>/    polls.views.DetailView  detail
/polls/<poll_id>/results/   polls.views.poll_results    results
/polls/<question_id>/vote/  polls.views.vote    vote
```

**Comment by smarzban** [comment]
@milton ready for review :)

**Comment by miltontony** [comment]
:+1:

**Pull request merged**